app/src/main/java/ch/epfl/cs311/wanderwave/ui/components/tracklist/TrackList.kt

```diff
@@ -46,16 +51,19 @@ fun TrackList(
         horizontalArrangement = Arrangement.SpaceBetween) {
           if (title != null) {
             Text(
-                text = title,
+                text = title.toLowerCase().replace("_", " "),
```

Comment:
Do we really want to have all songs titles in lower case ? I think we should try to stick as close as possible to the way Spotify displays it

Reply:
Actually, if you look at how the component tracklist is called, we do not modify the title of the song here, but the title of the track list (TOP_SONGS => top songs, LIKED_SONGS=> liked songs). I can maybe change that, but I think that it will be change to a way better maneer when we are going to look at the UI again, don't worry about it 😉

Reply:
You're right, simple misunderstanding on my part. Thanks for clarifying !